spec/loophole-spec.coffee

```diff
@@ -28,7 +28,47 @@ describe "Loophole", ->
       result = allowUnsafeNewFunction -> 42
       expect(result).toBe 42
 
+    it "works with async callback", ->
+      result = null
+
+      runs ->
+        allowUnsafeNewFunction (callback) ->
+          setTimeout(
+            ->
+              result = 42
```

Comment:
Could you actually perform the unsafe new in the the callbacks like the other tests do?